examples/models/llama2/evaluate/eager_eval.py

```diff
@@ -77,10 +79,21 @@ def tok_decode(self, tokens):
 
     def _model_call(self, inps):
         if self._use_kv_cache:
-            pos_tensor = torch.tensor([0], dtype=torch.int64, device=self.device)
-            # Batch process the whole sequence.
-            logits = self._model(inps[:, : self._max_seq_length], pos_tensor)
-            return logits
+            if not self._dynamic_shape:
```

Comment:
When do we not enable dynamic shape?

Reply:
I believe dynamic_shape is default to true now. We can probably ignore the `False` case here.